tests/test_import.py

```diff
@@ -488,7 +488,21 @@ def test_import_noChannelData(mocker, datadir):
     basedir = datadir.joinpath("xmlimport_noChannelData")
     with pytest.raises(RuntimeError) as excinfo:
         pyhf.readxml.parse(basedir.joinpath("config/example.xml"), basedir)
-        assert 'Channel channel1 is missing data. See issue #1911' in str(excinfo.value)
+    assert 'Channel channel1 is missing data. See issue #1911' in str(excinfo.value)
+
+
+def test_import_noChannelDataPaths(mocker, datadir):
+    _data = [0.0]
+    _err = [1.0]
+    mocker.patch('pyhf.readxml.import_root_histogram', return_value=(_data, _err))
+
+    basedir = datadir.joinpath("xmlimport_noChannelDataPaths")
```

Comment:
This duplicates the setup used for #1911 (other than a one line change to the xml) which isn't really ideal, but I figured it's not really better to copy the files to a temporary directory and edit the file there either.

Reply:
This is duplicating #1931, not #1911. It's better to just have this duplicated/hard-coded. I don't see why this is an issue.